docker-app/qfieldcloud/core/models.py

```diff
@@ -882,6 +881,11 @@ class Meta:
     data_last_updated_at = models.DateTimeField(blank=True, null=True)
     data_last_packaged_at = models.DateTimeField(blank=True, null=True)
 
+    repackaging_cache_expire = models.DurationField(
```

Comment:
```suggestion
    repackaging_cache_expire_m = models.DurationField(
```

Reply:
It's a duration field (stored as a postgres interval and converted to a python timedelta), so no units

Reply:
you are right, sorry, I forgot to check it is no longer a simple `int` field